assets/js/door.js

```diff
@@ -26,31 +26,31 @@ class View {
  */
 
 function getDoorId() {
-  const maybeDoorDom = Maybe(document.getElementById("door"))
+  const maybeDoorDom = Maybe(document.getElementById('door'))
   const maybeDoorId = maybeDoorDom.getAttribute('data-door-id')
 
-  return maybeDoorId.value;
+  return maybeDoorId.value
 }
 
 /* Public interface
  *
  */
 export default function(socket) {
-  const doorId = getDoorId();
-  if (!doorId) return;
+  const doorId = getDoorId()
+  if (!doorId) return
 
   const client = new Client(doorId)
   const view = new View(client)
 
   const channel = socket.channel(`door:${doorId}`, {})
 
   channel.join()
-    .receive("ok", resp => {
-      channel.push("ring", {client_id: client.id})
+    .receive('ok', _resp => {
+      channel.push('ring', {client_id: client.id})
     })
-    .receive("error", resp => { console.log("Unable to join", resp) })
+    .receive('error', resp => { console.log('Unable to join', resp) })
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)